Bugsnag/Payload/BugsnagEvent.m

```diff
@@ -339,6 +339,30 @@ - (instancetype)initWithKSCrashData:(NSDictionary *)event {
     NSDictionary *error = [event valueForKeyPath:@"crash.error"];
     NSString *errorType = error[BSGKeyType];
 
+    // Data coming from KSCrash will have either user.handledCount = 0 and user.unhandledCount > 0, or the opposite
+    BOOL wasUnhandled = [event valueForKeyPath:@"user.unhandledCount"] == nil ||
```

Comment:
I don't understand this logic... aren't all KSCrashes unhandled unless the developer has specifically called the following in their `onCrash` handler?

```
writer->addBooleanElement(writer, "unhandled", false);
```

Reply:
Note: handled errors (i.e. `Bugsnag.notify()`) go through `-initWithUserData:`

Reply:
I wasn't really sure if that's always the case. It felt safer to check that it was indeed originally marked unhandled.

Reply:
This initializer is only used for unhandled errors;

```
/**
 * Creates a BugsnagEvent from unhandled error JSON. Unhandled errors use
 * the JSON schema supplied by the KSCrash report rather than the Bugsnag
 * Error API schema, which is more complex to parse.
 *
 * @param event a KSCrash report
 *
 * @return a BugsnagEvent containing the parsed information
 */
- (instancetype)initWithKSCrashData:(NSDictionary *)event {
```